src/frontend/src/pages/Discover.tsx

```diff
@@ -1,4 +1,25 @@
+import { useEffect } from "react";
+import { useAppDispatch, useAppSelector, type RootState } from "../store/store";
+import { getNeighborhoodsAndBounds } from "src/features/neighborhoods/neighborhoodsThunk";
+
 const Discover = () => {
+    const dispatch = useAppDispatch();
+    const { loading, error, neighborhoods, neighborhoodBounds } =
+        useAppSelector(({ neighborhoods }: RootState) => neighborhoods);
+
+    useEffect(() => {
+        // Fetch initial neighborhoods data
+        const isNeighborhoodDataEmpty =
+            !neighborhoods?.features || !neighborhoodBounds?.features;
+        // TODO: Refactor on adding login workflow
+        const testAuthToken = import.meta.env.DEV
+            ? import.meta.env.VITE_DEVAUTHTOKEN
+            : import.meta.env.VITE_STGAUTHTOKEN;
```

Comment:
I wonder if a simpler (and also temporary) solution would be just making it `VITE_AUTHTOKEN`, and we supply different values in the dotenv files for the corresponding environments- thinking about what to use on staging after we merge this PR. What do you think?

Reply:
I like that, much simpler! Added a commit adjusting to use one `VITE_AUTHTOKEN` between environments (6f6a80b9a4affdac7a7f889d0e392b5f56b13833).

Reply:
Great. Now we will just need to update the dotnet file for staging.